v0.7/speech_recognition/rnnt/README.md

```diff
@@ -27,20 +26,10 @@ windows of size 20 milliseconds. Note that every three filterbanks are
 concatenated together ("feature splicing"), so the model's effective
 frame rate is actually 30 milliseconds.
 
-TODO: Figure out if preprocessing is considered part of the
-benchmark. For streaming inference, it definitely should be part of
-the benchmark. For training, it should not be part of the
-benchmark. For offline inference, it probably shouldn't be part of the
-benchmark, but I'm not sure. It depends upon whether your offline
-workload will be running more the one model on the inputs (e.g., voice
-activity detection, followed by actual speech-to-text. I am fairly
-skeptical that computing log mel spectra is particularly compute
-intensive.) Slide 11 here confirms that skipping featurization can be
-reasonable for offline use case:
-https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9672-accelerate-your-speech-recognition-pipeline-on-the-gpu.pdf It's not a bottleneck.
+No dithering takes place.
 
 ### Test data order
-In what order is the test data traversed? TODO
+TODO. Does this even matter for inference?
```

Comment:
@guschmue right now, I can run over the full dataset in single stream mode in 2-3 hours, depending on the computer (I developed on two computers, one with a network-attached storage which is higher latency). I'm guessing that this is too long for inference. What's a good length for an inference task to run?

Reply:
for single stream we run mostly for 60 seconds in performance mode on ~1000 samples.
accuracy model on imagenet is maybe 10 minutes and most of the time we spend getting the images into shape.

Reply:
Is it expected to run over the whole test dataset in accuracy mode?

Reply:
Yes, the accuracy mode will cover the entire test dataset. What is the 90th percentile latency in SingleStream?